### PR TITLE
Update to disable collecting PodLogs by default & rev Conformance versions

### PR DESCRIPTION
**Pull request by timothysc**
Due to reported experience #82

However I was not able to repro a bug that was reported there. 

Signed-off-by: Timothy St. Clair <timothysc@gmail.com>

### CODE REVIEW

**Comment by chuckha** [comment]
did you mean #82? #84 seems unrelated

**Comment by timothysc** [comment]
@chuckha yes.  updated.

File: examples/ksonnet/components/10-configmaps.jsonnet [hunk 1]
```diff
@@ -234,7 +233,7 @@ local e2eConfig = {
     containers: [
       {
         name: "e2e",
-        image: "gcr.io/heptio-images/kube-conformance:latest",
+        image: "gcr.io/heptio-images/kube-conformance:v1.8.0",
```
Comment (chuckha) [inline examples/ksonnet/components/10-configmaps.jsonnet]: How is this related to the PodLogs?

Comment (timothysc) [reply examples/ksonnet/components/10-configmaps.jsonnet]: I was testing latest changes, and in a similar area.  Just expedience.
